modules/ocf_mirrors/files/healthcheck

```diff
@@ -53,6 +57,19 @@ def get_updated_unix_timestamp(mirror_url):
     return datetime.utcfromtimestamp(int(req.text.split()[0]))
 
 
+def get_updated_recursive_ls(mirror_url):
+    """Find the time the host was last synced.
+    Should contain the output of ls -lR
+    >>> get_updated_unix_timestamp(mirror_url)
+    datetime.datetime(2018, 1, 24, 10, 3)
+    """
+    req = requests.get(mirror_url)
+    req.raise_for_status()
+    dates = [' '.join(line.split()[5:7]) for line in req.text.splitlines()
```

Comment:
I think a comment here with an example line would help a lot to show why you are getting these specific indices.